INSTALL.md

```diff
@@ -275,6 +276,84 @@ librehealthehr/sites/default/letter_templates
 
 If there are other files that you have customized, then you will also need to treat those carefully.
 
+##  Windows Setup
+
+#### Setup
+To run LibreEHR on Windows, [XAMPP](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) is needed with a compatible version
+
+**Note:** 
+
+1. Must have php [7.0](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/_) or [5.6](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/) ([5.6.30](https://sourceforge.net/projects/xampp/files/XAMPP%20Windows/5.6.30/) recommended)
+
+2. php 7.1x isn't supported
```

Comment:
Make the currently un supported. It will be when I can get you to fix it after GSI